ocaml/idl/datamodel_errors.ml

```diff
@@ -176,8 +172,8 @@ let _ =
     () ;
   error Api_errors.memory_constraint_violation_maxpin ["reason"]
     ~doc:
-      "The dynamic memory range violates constraint static_min <= dynamic_min \
```

Comment:
I think this change was actually a fix to the error (independent of disabling DMC), as static-min is always allowed to be less than the other three params.